base/gmp.jl

```diff
@@ -173,51 +172,50 @@ end
 for (op, T) in ((:fac_ui, Culong), (:set_ui, Culong), (:set_si, Clong), (:set_d, Cdouble))
     op! = Symbol(op, :!)
     @eval begin
-        $op!(x::BigInt, a) = (ccall($(gmpz(op)), Void, (mpz_t, $T), &x, a); x)
+        $op!(x::BigInt, a) = (ccall($(gmpz(op)), Void, (Mpz_t, $T), x, a); x)
         $op(a) = $op!(BigInt(), a)
     end
 end
 
-popcount(a::BigInt) = ccall((:__gmpz_popcount, :libgmp), Culong, (mpz_t,), &a) % Int
+popcount(a::BigInt) = ccall((:__gmpz_popcount, :libgmp), Culong, (Mpz_t,), a) % Int
```

Comment:
Unrelated but `% Int` seems dangerous here because of the `Culong` . Would be better to wrap in `Int`, even if a bit slower (which shouldn't be a concern because `BigInt`'s are not used for performance). I can open a different PR for this.